src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner
+  = person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+
+      <p className="Person__partner">
+        {person.isMarried ? partner : 'I am not married'}
```

Comment:
Move this logic to the variable and use it here

Reply:
Do you mean logic of person age or person partner?